### ISSUE TITLE
Any interest in supporting travis?

### ISSUE THREAD

**Issue by dakkusingh** [deviantintegral/drupal_tests#40]
It would be great to have the flexibility to use travis for module testing, the same way as https://github.com/Lullabot/drupal8ci supports drupal testing with Travis.

Is there interest in adding this feature? Are there any pros/cons over the 2 ci solutions for module testing?

**Comment by deviantintegral** [comment]
Yes, that would be great! @juampynr has been working towards a docker-compose solution for Travis, which would let us reuse most of the circleci work too.

**Comment by dakkusingh** [comment]
Just seen @juampynr recent push with Travis.
https://github.com/Lullabot/drupal8ci/blob/master/dist/travisci/.travis/docker-compose.yml

In some ways I quite like that it is using Wodby images. I also use wodby for production stack, and their dinky image size is always a plus (Alpine).

It does however mean that:
1) should we re-jig drupal_tests to use wodby instead of drupal-apache (official docker) images? Wodby uses Alpine and therefore images are much smaller.
2) should we re-jig circle ci flow to closely align with travis?

Needs some food for thought!! What do you think?

**Comment by dakkusingh** [comment]
crosspost: https://github.com/Lullabot/drupal8ci/issues/22
